tutorials/plugins/gdnative/files/cpp_example/SConstruct

```diff
@@ -22,6 +22,9 @@ cpp_library = "libgodot-cpp"
 # only support 64 at this time..
 bits = 64
 
+# Make 'demo/bin' directory
+Execute(Mkdir('demo/bin'))
```

Comment:
Should be `env['target_path']`, since this is configurable as seen above.
But it shouldn't be needed in the first place, SCons should take care of creating the needed path for its build artifacts automatically. Since later on the platform name is appended to `target_path`, and there is no `Mkdir` call, it seems to show that it works fine to create those subdirectories.